components/UserList/index.js

```diff
@@ -1,21 +1,108 @@
+const styless = {
+  root: {
+    flexGrow: 1,
+  },
+  'list-expand': {
+    border: '1px solid #0288D1',
+  },
+  Word_Coler: {
+    color: '#2196F3',
+  },
+  'userFullName-before': {
+    'font-size': '0.875rem',
+    color: '#2196F3',
+  },
+  'userFullName-after': {
+    'font-size': '1.2em',
+    color: '#212121',
+  },
+};
+class UserList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bgColor: {
+        start: Object.keys(BlueColor).indexOf('200'),
+        end: Object.keys(BlueColor).indexOf('0'),
+      },
+    };
+  }
+getFullName = claim => `${claim.firstName} ${claim.lastName}`
+listExpand = panelIndex => (event, expanded) => {
+  const parent = event.currentTarget.parentElement;
+  let {start} = this.state.bgColor;
+  const {end} = this.state.bgColor;
+  let operator = 0;
+
+  if (start - end < 0) {
+    operator = end - start;
+    start = end;
+  } else {
+    operator = start - end;
+  }
+
+  operator += 1;
+  const colorIndex = Object.keys(BlueColor)[start - (panelIndex % operator)];
+
+  parent.style['background-color'] = BlueColor[colorIndex];
+
+  if (expanded) {
+    parent.style.border = styless['list-expand'].border;
+    parent.querySelector('p.userFullName').style.fontSize = styless['userFullName-after']['font-size'];
+    parent.querySelector('p.userFullName').style.color = styless['userFullName-after'].color;
+  } else {
+    parent.setAttribute('style', '');
+    parent.querySelector('p.userFullName').style.fontSize = styless['userFullName-before']['font-size'];
+    parent.querySelector('p.userFullName').style.color = styless['userFullName-before'].color;
+  }
+}
+render() {
   return (
-    <div>
-      {users.map(user => (
-        <div key={user.id}>
-          <Link href={`/user?id=${user.id}`} as={`/users/${user.id}`}>
-            <a>{user.id}</a>
-          </Link>
-        </div>
-      ))}
+    <div style={styless.root}>
+      {
+        this.props.users.map((user, i) => (
+          <ExpansionPanel onChange={this.listExpand(i)}>
+            <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+              <Typography className="userFullName" style={styless.Word_Coler}>{this.getFullName(user.claim)}</Typography>
+            </ExpansionPanelSummary>
+            <ExpansionPanelDetails>
+              <Typography>
+                <div style={styless.chatroom}>
+                  <Link href={`/user?id=${user.id}`} as={`/users/${user.id}`}>
+                    <IconButton tooltip="Profile">
+                      <Badge
+                        secondary
+                        badgeStyle={{ top: 12, right: 12 }}
+                      >
+                        <AccountBoxIcon />
+                      </Badge>
+                    </IconButton>
+                  </Link>
+                </div>
+
+              </Typography>
+            </ExpansionPanelDetails>
+          </ExpansionPanel>
+  ))
+  }
     </div>
   );
-};
+}
+}
```

Comment:
Because you are using the fat arrow with class method, ESLint cannot detect the indent problem in `render` method.

Please fix the indent problem in `render` method. 😏

Reply:
We need explicit guidelines along with code examples.